src/Client.php

```diff
@@ -16,9 +16,9 @@
 use InvalidArgumentException;
 use IteratorAggregate;
 use Predis\Command\CommandInterface;
+use Predis\Command\Container\ContainerFactory;
+use Predis\Command\Container\ContainerInterface;
 use Predis\Command\RawCommand;
-use Predis\Command\Redis\Container\ContainerFactory;
-use Predis\Command\Redis\Container\ContainerInterface;
```

Comment:
This is a breaking change, eh?

Reply:
Files was just moved from one directory to another to make it more consistent, and this functionality was just added within v2.1.2

Reply:
It's okay, I revert changes so it's not breaking anymore 👌